### PR TITLE
update .gitignore & remove Thumbs.db

### PR DESCRIPTION
**Pull request by tgfjt**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 out
-node_modules
\ No newline at end of file
+node_modules
+npm-debug.log
+Thumbs.db
+node_modules/
```
Comment (jednano) [inline .gitignore]: You shouldn't need to specify `node_modules` twice.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,5 @@
 out
-node_modules
\ No newline at end of file
+node_modules
+npm-debug.log
+Thumbs.db
```
Comment (jednano) [inline .gitignore]: This should be in your global `.gitconfig`.

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,5 @@
 out
-node_modules
\ No newline at end of file
+node_modules
+npm-debug.log
```
Comment (jednano) [inline .gitignore]: `*.log` would be more comprehensive.

File: .gitignore (hunk 2 above)
Comment (tgfjt) [inline .gitignore]: Might be so, but with reference to:
https://github.com/Microsoft/vscode/blob/master/.gitignore

File: .gitignore (hunk 1 above)
Comment (tgfjt) [inline .gitignore]: Ooops my bad.

**Comment by tgfjt** [comment]
@jedmao 
Thanks for your reviewing. I fixed & pushed.
I'm just concerned about Thumbs.db: https://github.com/Microsoft/vscode-editorconfig/blob/master/Thumbs.db
Do you think that I could remove it?

**Comment by jednano** [comment]
Just add Thumbs.db to your global .gitignore and it won't ask you to commit it.

**Comment by jednano** [comment]
But yeah, the Thumbs.db file should not be uploaded to this repo.

**Comment by jednano** [comment]
Yes, remove it.

**Comment by tgfjt** [comment]
Thanks.
just removed `Thumbs.db`.

**Comment by jednano** [comment]
:+1:

**Comment by chrisdias** [comment]
i fixed the Thumbs.db problem and updated .gitignore, but i didn't add .log to .gitignore. What is the purpose of that entry?

**Comment by jednano** [comment]
@chrisdias, sometimes running `npm` commands can produce errors in the form of `npm-debug.log`. Other tools too, I'm sure. The `*.log` ensures they aren't accidentally committed to the repo.

**Comment by chrisdias** [comment]
merged

**Comment by tgfjt** [comment]
Thanks:smile:

**Pull request merged**